meinberlin/apps/budgeting/assets/helpers.js

```diff
@@ -14,7 +14,12 @@ const months = [
   django.gettext('November'),
   django.gettext('December')
 ]
+
 export const toDate = (isodate) => {
   const d = new Date(isodate)
   return `${d.getDate()}. ${months[d.getMonth()]} ${d.getFullYear()}`
 }
+
+export const intComma = (number) => {
+  return Number(number).toLocaleString()
```

Comment:
is this adding a comma for cents? as we can only have whole numbers do we need it?

Reply:
maybe in js files we should consider having comments? as can't always tell, specially with no fuile name to help

Reply:
@phillimorland actually it is for the thousand separator (20.000 or 20,000). Works as the filter in django. I admit the name could be better but i named it how the template filter is named in django.

Reply:
fab!